test-harness/specs/E2E-Accept-Content-Negotiation-AC-4.oas2.txt

```diff
@@ -15,14 +15,14 @@ paths:
         200:
           description: Get Todo Items
           examples:
-            application/json: "hello"
+            application/json: '"hello"'
```

Comment:
In case you're asking why this is happening — strings are not JSONS — they need to be quoted.